foglamp.version

```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```

Comment:
@Singhal-Vaibhav Missing service_notification.version. Please add.

Here we have foglamp.version as well, is this really required @pintomax @MarkRiddoch
I think we are NOT keeping foglamp.version file in rule- plugins; as direct dependency is notification service which has dependency check for foglamp version

Reply:
@praveen-garg we need both foglamp.version and  service_notification.version.

foglamp.version is being used by make_rpm and make_deb, any plugin.
service_notification.version is taken only for notification-rule and notification-delivery plugins

Reply:
service_notification.version should certainly be added. As for having foglamp_version, it does no harm, and it does allow for the plugin to depend on a feature in FogLAMP that the service does not use.